core/frontend/src/components/vehiclesetup/viewers/VehicleViewer.vue

```diff
@@ -14,8 +14,8 @@ import autopilot from '@/store/autopilot_manager'
 export default Vue.extend({
   name: 'VehicleViewer',
   components: {
-    Sub: () => import('./SubViewer.vue'),
-    Rover: () => import('./RoverViewer.vue'),
+    Sub: () => import('./GenericViewer.vue'),
+    Rover: () => import('./GenericViewer.vue'),
     None: () => import('./NoneViewer.vue'),
```

Comment:
should we move None to generic as well ? And make it take care of everything ?

Reply:
done